solidity/dashboard/src/components/OwnedTokensOverview.jsx

```diff
@@ -33,7 +38,7 @@ const OwnedTokensOverview = ({ keepBalance, stakedBalance }) => {
           <h5 className="text-grey-70">staked</h5>
           <h4 className="text-grey-70">{displayAmount(stakedBalance)}</h4>
```

Comment:
do we want to display formatted amount here as well?

Reply:
I did it according to Figma.